### PR TITLE
Translations/home

### PR DESCRIPTION
**Pull request by vyvil**
PR applying translations to Home screen and related components

### CODE REVIEW

**Review by IAmKio — APPROVED** [review]
Great work as always!

File: src/screens/Home/Home.js [hunk 1]
```diff
@@ -324,15 +325,19 @@ class HomeScreen extends React.Component<Props> {
         cornerIcon={symbol === DAI ? daiIcon : usdcIcon}
         itemImageRoundedSquare
       >
-        <BaseText fontSize={fontSizes.big} primary>{userTickets} tickets</BaseText>
-        <BaseText secondary>{winChance} chance</BaseText>
+        <BaseText fontSize={fontSizes.big} primary>
+          {t('poolTogetherContent.label.ownedTickets', { count: userTickets })}
+        </BaseText>
+        <BaseText secondary>
+          {t('poolTogetherContent.label.winChance', { chancePresent: winChance })}
```
Comment (poocart) [inline src/screens/Home/Home.js:332]:
```suggestion
          {t('poolTogetherContent.label.winChance', { chancePercent: winChance })}
```

**Review by poocart — APPROVED** [review]

**Review by poocart — APPROVED** [review]
All good but small comment 👍

File: src/screens/Home/Home.js (hunk 1 above)
Comment (vyvil) [reply src/screens/Home/Home.js:332]: Yikes. Thanks for spotting it!

**Pull request merged**